app/Jobs/ImportFileRecords.php

```diff
@@ -91,13 +92,12 @@ public function handle(Rogue $rogue)
         info('STARTING '.$this->importType.' IMPORT', ['options' => print_r($this->importOptions, true)]);
 
         $records = $this->getRecords($this->filepath);
-        $user = \Auth::user();
 
         $importFile = new ImportFile([
             'filepath' => $this->filepath,
             'import_type' => $this->importType,
             'row_count' => $this->totalRecords,
-            'user_id' => $user->northstar_id,
+            'user_id' => $this->user ? $this->user->northstar_id : null,
```

Comment:
Also, strange that this isn't working when logged in 🤔 would `Auth::id()` work perchance? I guess presumably not if `Auth::user()` isn't...

Reply:
Calling `Auth::user()` from this job doesn't work because the job is being processed outside of the web session, so it's technically just Chompy doing all these actions at this point. Now that it's being serialized into the job, I think it's unlikely that it'd ever be empty...

Reply:
It's slightly getting ahead of myself - but this could be null if we support automating downloading CSV's from the Rock the Vote API (it also got me wondering about calling this something like `created_by` vs `user_id`, but figured a null `user_id` would suffice...)

Reply:
Ooooh! Forward-thinking! 👍